Userland/Libraries/LibWeb/Layout/TextNode.cpp

```diff
@@ -42,7 +52,7 @@ void TextNode::compute_text_for_rendering(bool collapse)
     }
 
     if (!collapse || data.is_empty()) {
-        m_text_for_rendering = data;
+        m_text_for_rendering = apply_text_transform(data, computed_values().text_transform());
```

Comment:
At `auto& data = dom_node().data();`, above, you could do the transform there, rather than multiple calls.

Reply:
Great idea :)